common/segments/root.xml

```diff
@@ -341,9 +343,10 @@ For terms of use, see http://www.unicode.org/copyright.html
 				<variable id="$CR">\p{Word_Break=CR}</variable>
 				<variable id="$LF">\p{Word_Break=LF}</variable>
 				<variable id="$Newline">\p{Word_Break=Newline}</variable>
-				<variable id="$Extend">\p{Word_Break=Extend}</variable>
+				<variable id="$Extend">[\p{Word_Break=Extend}[🏻-🏿 ำ ຳ]]</variable>
 				<!-- Now normal variables -->
-				<variable id="$Format">\p{Word_Break=Format}</variable>
+				<variable id="$Prepend">\p{Grapheme_Cluster_Break=Prepend}</variable>
+				<variable id="$Format">[\p{Word_Break=Format}-\p{Grapheme_Cluster_Break=Prepend}]</variable>
```

Comment:
Neither UAX#29 nor ICU word.txt have the chnages above, where are they coming from?